docs/core/eventdata/tracks.md

```diff
@@ -1,4 +1,730 @@
+- jacobian
+
+To illustrate why this can be useful, consider again {numref}`ckf_tree`, where
+$S_2$ and $S_3$ branch out from a shared $S_1$. In this case, the predicted
+parameter vector and covariance, as well as the jacobian from $S_1\to S_2$ and
+$S_1 \to S_3$ will be identical. In this case, the combinatorial track finding
+will use the sharing functionality to share these components.
+
+:::{attention}
+Sharing these components introduces *cross-talk* between track states, and this
+is intentional.  If e.g. the predicted covariance is modified through either of
+the track states, the changes will be visible when accessed from the other
+track state as well.
+:::
+
+(track_edm_dynamic_columns)=
+## Dynamic columns
+
+Aside from the static properties that both the track states and the track, the
+EDM supports adding almost arbitrary additional information as dynamic columns.
+The implementation of the dynamic column mechanism is given by the backend,
+where the interface layer classes {class}`Acts::MultiTrajectory` and
+{class}`Acts::TrackContainer` and associated proxies only coordinate the
+creation, access and copying of dynamic columns.
+
+The following illustrates the
+usage of dynamic columns for {class}`Acts::TrackContainer`, but usage on
+{class}`Acts::MultiTrajectory` is identical.
+
+Assume you create a track container using some combination of backends (see
+[](#edm_track_backends) for information on the backends shipped with ACTS).
+
+```cpp
+Acts::TrackContainer tc{/*...*/};
+
+// add dynamic columns programmatically
+
+tc.addColumn<float>("col_a");
+tc.addColumn<uint8_t>("col_b");
+
+```
+
+Adding columns is only supported on *mutable* track containers, const track
+containers should contain the original dynamic columns from when they were
+created. It is up to the backend to implement recovering dynamic columns from
+e.g. input files.
+
+:::{note}
+Which types are supported depends on the backend being used. See
+[](#edm_track_backends) for information on the backends shipped with ACTS, and
+which types they support.
+:::
+
+With these dynamic columns registered, it is now possible to set and get values for these columns on tracks.
+
+```cpp
+using namespace Acts::HashedStringLiterals;
+auto track = tc.getTrack(tc.addTrack());
+
+// these two are equivalent
+track.component<float, "col_a"_hash>() = 42.42;
+track.component<float>("col_a"_hash) = 52.52;
+std::cout << track.component<float, "col_a"_hash>() << std::endl; // prints: 52.52
+```
+
+:::{tip}
+The expression `"col_a"_hash` is a user-defined literal that internally calls
+```cpp
+Acts::hashedString("col_a");
+```
+This literal is only available after
+```cpp
+using namespace Acts::HashedStringLiterals;
+```
+:::
+
+The components are accessed by a hash of the name of the component. This hash
+can be calculated from a string at compile-time, if the string is known at
+compile time.  The difference between the two component access signatures is
```

Comment:
```suggestion
compile time. The difference between the two component access signatures is
```
sorry about picking each of those but they jump into my eye 😄 I guess in the display it does not matter

Reply:
They result from the vim reflow that I do. I tried to remove all of them now.